package com.shadov.codehellven.api.code.domain.exception

internal class CodeExecutionException : RuntimeException("Exception occured during code execution")